apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -30,6 +30,7 @@ const validators = [
   {
     delegate: new Delegate(Account.SOL_2, "0.001", "Ledger by Chorus One"),
     xrayTicket: "B2CQA-2730, B2CQA-2764",
+    bugTicket: "LIVE-14500",
```

Comment:
I would leave this as a ToDo comment in the code since we commented out code which made these tests fail